location_names: 
  - "L1"
  - "L2"
#  - "L3"
#  - "L4"
#  - "L5"

L1: { 
  default: {
    x: 0.40,
    y: 0.20,
    z: 0.09, 
  },
  object_classes: [
    small
  ],
  small: {
    x: 0.40,
    y: 0.20,
    z: 0.04, 
  },
  detection_radius: 0.20, 
  orientation_type: up_x
}

L2: { 
  default: {
    x: 0.40,
    y: -0.20,
    z: 0.09, 
  },
  object_classes: [
    small
  ],
  small: {
    x: 0.40,
    y: -0.20,
    z: 0.03, 
  },
  detection_radius: 0.20, 
  orientation_type: up_x
}

#L2: { 
#  x: 0.40,
#  y: -0.20,
#  z: 0.09, 
#  r: 0.20, 
#}
#
#L3: { 
#  x: -0.40,
#  y: 0.20,
#  z: 0.09, 
#  r: 0.20, 
#}
#
#L4: { 
#  x: -0.40,
#  y: -0.20,
#  z: 0.09, 
#  r: 0.20, 
#}
#
#L5: { 
#  x: 0.40,
#  y: 0.00,
#  z: 0.09, 
#  r: 0.20, 
#}
#
#
#location_orientation_types:
#  - up_x
#  - up_x 
#  - up_x
#  - up_x 
#  - up_x